rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -1,11 +1,36 @@
 = Managing Bots
 
-Bots can be licensed to use secure sessions. Some properties, like name, service times, and phase permissions can be edited. Deinstalled bots can be removed from RPA Manager.
+Newly installed bots connect to RPA Manager automatically.
+
+As a bot administrator, you can manage the following properties of a bot:
+
+* Name and describe a bot.
+* Assign service times to a bot to determine their office hours.
+* Restrict bots to project phases
+* Set the maximum number of sessions a bot may use.
+
+If a bot is troubling you, you can do the following:
+
+* Download the log files.
+* Restart the bot.
+* Delete the bot.
 
 == Before You Begin
 
 * At least one bot must be installed on a separate computer.
-* If you want to assign more than one secure session to a bot, make sure that a terminal server software is installed on the computer of the bot.
+* On the computer where you want to run automations and where a bot therefore is installed, check the system data:
```

Comment:
See previous similar edits. Opportunity for reuse?

Reply:
This needs more consideration (= too few time before release) --> Included in Doc Space Redesign project